apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -99,14 +104,10 @@ def create(
             attrs=attrs,
             sparse=True,
             allows_duplicates=False,
-            offsets_filters=[
-                tiledb.DoubleDeltaFilter(),
-                tiledb.BitWidthReductionFilter(),
-                tiledb.ZstdFilter(),
-            ],
-            capacity=100000,
-            cell_order="row-major",
-            tile_order="row-major",
+            offsets_filters=create_options.offsets_filters(),
+            capacity=create_options.get("capacity", 100000),
+            cell_order=create_options.get("cell_order", "row-major"),
+            tile_order=create_options.get("tile_order", "row-major"),
```

Comment:
will TileDB-Py ignore the `tile_order` value when `cell_order = "hilbert"`?  Or do we have to choose a default based upon `cell_order`?   (I think it will ignore, but have not confirmed)

Reply:
@bkmartinjr we need to say `tile_order=None` explicitly (I know because I've tried :) )

Reply:
As long as we can do that via this API, all is well.  I just know we will end up trying hilbert at some point

Reply:
I've updated the behavior here to use the default only if *both* `cell_order` and `tile_order` are unset.

So:

`{"cell_order": "x"}` = `cell_order = "x"`, `tile_order = None`
`{"tile_order": None}` = `cell_order = None`, `tile_order = None`
`{}` = `cell_order = "row-major"`, `tile_order = "row-major"`